src/sitemap.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    <url><loc>http://dex.software/home</loc><priority>0.5</priority></url>
+    <url><loc>http://dex.software/home/project/overview</loc><priority>0.5</priority></url>
+    <url><loc>http://dex.software/home/project/add/source</loc><priority>0.5</priority></url>
+    <url><loc>http://dex.software/home/account/login</loc><priority>0.5</priority></url>
```

Comment:
These should refer to the canonical URLs on DeX, which are all secure (https), these URLs all redirect.